Sources/VaporAWSLambdaRuntime/LambdaServer.swift

```diff
@@ -142,9 +144,6 @@ public class LambdaServer: Server {
     }
 
     public func shutdown() {
-        // this should only be executed after someone has called `app.shutdown()`
-        // on lambda the ones calling should always be us!
-        // If we have called shutdown, the lambda server already is shutdown.
-        // That means, we have nothing to do here.
+        self.lambdaLifecycle.shutdown()
```

Comment:
@gwynne @0xTim this code never get's called, even though it is part of the `Server` protocol